Lib/test/test_dis.py

```diff
@@ -683,16 +683,10 @@ def test_boundaries(self):
 
     def test_widths(self):
         for opcode, opname in enumerate(dis.opname):
-            if opname in ('BUILD_MAP_UNPACK_WITH_CALL',
-                          'BUILD_TUPLE_UNPACK_WITH_CALL',
+            if opname in (
```

Comment:
Any reason why we're not making space for the others? It would be nice if we could just get rid of these special-cases.

Reply:
I wasn't sure we want to make it that large (would need to be 29). But we could.

Reply:
I personally think it's fine.

If 29 is too large, it means we need shorter opnames, not that we should break the table formatting. 😉